src/components/Story/StoryFull.js

```diff
@@ -312,26 +312,29 @@ class StoryFull extends React.Component {
               </span>
             </Tooltip>
             { Math.ceil(readingTime(post.body).minutes) > 1 &&
-              <Tooltip
-                title={
-                  <span>
+              <div className="inline-block">
+                <span className="CommentFooter__bullet" />
```

Comment:
I think we should change the class name. The name `CommentFooter_..` is not relevant here.